katastertools/sql/popisne-data.sql

```diff
@@ -23,7 +23,7 @@ BEGIN;
 		oce integer,
 		cas character varying(30),
 		crc integer,
-		naz_suboru character varying(20)
```

Comment:
Preco si menila naz_suboru na subor ? Nebolo potrebne to aktualizovat aj niekde inde v kode ?